packages/components/package.json

```diff
@@ -68,7 +68,6 @@
     "rollup": "^2.79.1",
     "rollup-plugin-multi-input": "^1.4.1",
     "rollup-plugin-postcss": "^4.0.2",
-    "ts-node": "^8.1.0",
```

Comment:
Can this move? I thought we just installed this here for the icon work recently: https://github.com/GetJobber/atlantis/commit/563b402a21e4c9e72142fe254f469b45a758cc0a

Reply:
I uninstalled it from `packages/components` because we're now running it from the root. It will not conflict with the changes from the icon work as it was installed in the root `package.json`

Reply:
Doh! I see now that this is the `components/package.json` -- ok, makes sense!